module/VuFind/src/VuFind/Config/YamlReader.php

```diff
@@ -110,13 +111,14 @@ public function get($filename, $useLocalConfig = true, $forceReload = false)
     /**
      * Given core and local filenames, retrieve the configuration data.
      *
-     * @param string $defaultFile Full path to file containing default YAML
-     * @param string $customFile  Full path to file containing local customizations
+     * @param string  $defaultFile    Full path to file containing default YAML
+     * @param string  $customFile     Full path to file containing local customizations
      * (may be null if no local file exists).
+     * @param boolean $useLocalConfig Use local configuration if available
```

Comment:
This is a problem throughout this file, not just in new code, but we generally use `@param bool` rather than `@param boolean`. I've opened #4931 to clean up existing type issues and detect future issues, but we should also fix the new ones introduced here.

```suggestion
     * @param bool $useLocalConfig Use local configuration if available
```